upgrade_test.py

```diff
@@ -1438,6 +1442,37 @@ def _update_argus_upgraded_version(self, node, new_version):
         except Exception as exc:  # pylint: disable=broad-except
             self.log.exception("Failed to save upgraded Scylla version in Argus", exc_info=exc)
 
+    def validate_limited_voters_feature_enabled(self):
+        InfoEvent(message="Check that limited voters feature enabled after upgrade for all nodes")
+        feature_state_per_node = []
+        for node in self.db_cluster.nodes:
+            result = wait_for(func=self.db_cluster.is_features_enabled_on_node,
+                              timeout=60,
+                              step=1,
+                              text=f"Check feature enabled on node {node.name}",
+                              throw_exc=False,
+                              feature_list=["GROUP0_LIMITED_VOTERS"],
+                              node=node)
+            feature_state_per_node.append(result)
+        if all(not enabled for enabled in feature_state_per_node):
+            InfoEvent(message="Limited voters feature is not supported or enabled on all nodes").publish()
+            return
+        elif all(feature_state_per_node):
+            InfoEvent(message="Limited voters feature enabled on all nodes").publish()
+            number_of_nodes = len(self.db_cluster.nodes)
+            num_of_non_voters = len(self.db_cluster.nodes[0].raft.get_group0_non_voters())
+            if number_of_nodes <= 5 and num_of_non_voters > 0:
+                InfoEvent(message='Number of voters are less than expected',
+                          severity=Severity.ERROR).publish()
+            elif number_of_nodes > 5 and num_of_non_voters == 0:
+                InfoEvent(message='Number of voters higher than expected',
```

Comment:
I think it would be informative if you wrote how many were expected and how many we have got. It might help debugging if this ever happens

Reply:
fixed